src/resources/postcss/views/_loader.pcss

```diff
@@ -0,0 +1,24 @@
+.tribe-events-container {
+	position: relative;
+
+	.tribe-events-view-loader {
+		align-content: center;
+		justify-content: center;
+		display: flex;
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		z-index: 25000;
```

Comment:
Let's use a sensible `z-index` value here, rather than a random high number. We can create a `z-index` variables file in `tribe-common-styles` if needed so we're being consistent.

Reply:
What would be a sensible `z-index` here? Since we want this to be on top of everything and customers might have other stuff running around?

Reply:
Added those variables; https://github.com/moderntribe/tribe-common-styles/pull/7